src/ts/ConvivaAnalytics.ts

```diff
@@ -34,6 +34,14 @@ export interface ConvivaAnalyticsConfiguration {
    * user agent string parsing by the Conviva SDK. (default: WEB)
    */
   deviceCategory?: Conviva.Client.DeviceCategory;
+
+  deviceBrand?: string;
+  deviceManufacturer?: string;
+  deviceModel?: string;
+  deviceType?: Conviva.Client.DeviceType;
+  deviceVersion?: string;
+  operatingSystemName?: string;
+  operatingSystemVersion?: string;
```

Comment:
Not sure if we should add a little documentation about those properties as well. Like we do it for `deviceCategory` 🤔

Reply:
Updated.